.github/workflows/tests.yml

```
@@ -29,7 +28,7 @@ jobs:
             testbench: 8.*
             carbon: ^2.63
```

Rather than make these other changes, can you try reverting and changing this to:

```suggestion
            carbon: ^2.67
```

I'd rather keep this testing if we can.